app/policies/course_lesson_policy.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class CourseLessonPolicy < ApplicationPolicy
+  def initialize(user, record)
+    @user = user
+    @record = record
+  end
+
+  def show?
+    true
+  end
+
+  def create?
+    admin_only
+  end
+
+  def edit?
+    admin_only
+  end
+
+  def update?
+    admin_only
+  end
+
+private
+
+  def admin_only
```

Comment:
Don't need to redefine this here, we get this by inheriting from `ApplicationPolicy`

Reply:
We redefine the admin_only slightly in the course_lesson policy because we need non-users to be able to access the content. It better just to put that in the ApplicationPolicy?

def admin_only
    return true if user&.admin?

Reply:
Ah sorry, hadn't spotted the difference between the two. Yes, I think I would just update ApplicationPolicy